portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACApi.macro

```diff
@@ -0,0 +1,95 @@
+definition {
+
+	macro createNewExportSchedule {
+		var curl = '''
+			http://localhost:8080/api/reports/export/${type}\?fromDate=${startDate}T${startTime}Z\&toDate=${endDate}T${endTime}Z
```

Comment:
Hey @christianstokes, I tested these cases using my local environment with my IP and sending localhost in the API curl and this case was passing using localhost or with the IP. I preferred to leave it with localhost so any change to the macro of getting the IP doesn't affect our tests, I believe the test would work on CI using localhost, but if you prefer I can change it. What do you think it's better?

Reply:
One way we can get around this issue is copying the root endpoint url where we generate the token. It would have the AC URL as well as the correct port

Reply:
I'm going to run one of the tests on CI so we can check the result as you suggested 😄

Reply:
Hey guys, [I ran the test on CI](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1853910984) and it only passed when I put curl to use with IP. Thank you very much for the suggestion, I had tried to run it before opening the PR, but because of the Bug, the test did not run completely.

`var analyticsCloudURL = PropsUtil.get("analytics.cloud.url");`

Get the correct IP and port of the AC

cc: @christianstokes @joshitagaki

Reply:
Awesome glad it worked @ViniLopes20!